oak/server/oak_runtime.cc

```diff
@@ -108,6 +110,39 @@ grpc::Status OakRuntime::Initialize(const ApplicationConfiguration& config) {
   return grpc::Status::OK;
 }
 
+std::string OakRuntime::NextNodeName(const std::string& contents) {
+  absl::MutexLock lock(&mu_);
+  int index = next_index_[contents]++;
+  return absl::StrCat(contents, "-", index);
+}
+
+bool OakRuntime::CreateWasmNode(const std::string& contents, std::unique_ptr<ChannelHalf> half,
```

Comment:
```suggestion
bool OakRuntime::CreateWasmNode(const std::string& wasm_module_name, std::unique_ptr<ChannelHalf> half,
```

WDYT?

Reply:
Prefer not to, as the name would need to change back once the later commits arrive.

Reply:
LGTM, but I note that the method name will have to change anyways, right?